test/service/data/pfb_manifest.results.json

```diff
@@ -806,6 +807,7 @@
             "document_id": [
                 "e8642221-4c2c-4fd7-b926-a68bce363c88"
             ],
+            "estimated_cell_count": "",
```

Comment:
I assume this is a substitute for `null`. Is there precedent for nullable numeric fields?

Reply:
Yes. `estimated_cell_count` in `projects` has a `null_int` field type like `total_estimated_cells` in `cell_suspensions`. 

Cell suspension [377f2f5a-4a45-4c62-8fb0-db9ef33f5cf0](https://github.com/DataBiosphere/azul/blob/0a40364598d1de4a7b53f9ba36b63a0169a171ee/test/indexer/data/587d74b4-1075-4bbf-b96a-4d1ede0481b2.metadata.json#L2) has no `total_estimated_cells` field, resulting in an empty string value for `total_estimated_cells` under [cell_suspensions_377f2f5a-4a45-4c62-8fb0-db9ef33f5cf0](https://github.com/DataBiosphere/azul/blob/0a40364598d1de4a7b53f9ba36b63a0169a171ee/test/service/data/pfb_manifest.results.json#L396)